corehq/apps/accounting/models.py

```diff
@@ -1754,6 +1754,13 @@ def is_below_minimum_subscription(self):
         else:
             return False
 
+    def user_can_change_subscription(self, user):
+        if not self.account.is_customer_billing_account:
+            return True
```

Comment:
is there a reason for using `not` here instead of just keeping it simple
```
if self.account.is_customer_billing_account:
    return False
```

assuming that if its a custom billing account, user can't change subscription.